### PR TITLE
Fix typos in docs

### LINKED ISSUE #9691: Grammatical errors in docs
Small grammatical errors are present in some of the files of the documentation.

### PR DESCRIPTION
**Pull request by sakkshm26**
## Hey, I just made a Pull Request!

Fixed some grammatical errors in the docs.
This PR will fix #9691. 

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by jhaals — APPROVED** [review]
Thanks a lot for the fixes @sakkshm26!

Just a note that the issue you created is not necessary prior to smaller changes like these.

File: GOVERNANCE.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ To become a maintainer you need to demonstrate the following:
 - Monitor Discord (delayed response is perfectly acceptable).
 - Triage GitHub issues and perform pull request reviews for other maintainers and the community.
 - Triage build issues - file issues for known flaky builds or bugs, and either fix or find someone to fix any master build breakages.
-- During GitHub issue triage, apply all applicable ([labels](https://github.com/backstage/backstage/labels)) to each new issue. Labels are extremely useful for future issue follow up. Which labels to apply is somewhat subjective so just use your best judgment. A few of the most important labels that are not self explanatory are:
+- During GitHub issue triage, apply all applicable ([labels](https://github.com/backstage/backstage/labels)) to each new issue. Labels are extremely useful for future issue to follow-up. Which labels to apply is somewhat subjective so just use your best judgment. A few of the most important labels that are not self-explanatory are:
```
Comment (freben) [inline GOVERNANCE.md:27]:
```suggestion
- During GitHub issue triage, apply all applicable ([labels](https://github.com/backstage/backstage/labels)) to each new issue. Labels are extremely useful for future issue follow-up. Which labels to apply is somewhat subjective so just use your best judgment. A few of the most important labels that are not self-explanatory are:
```

Comment (freben) [reply GOVERNANCE.md:27]: "issue follow-up" is used as a noun in and of itself, so either "to" should be taken out, or rewrite as "useful for knowing what issues to follow up on" or similar. But just taking out "to" is probably easiest.

File: GOVERNANCE.md [hunk 2]
```diff
@@ -71,9 +71,9 @@ The following is a non-exhaustive list of types of change, for which a reviewer
 - Additions and removals of entire packages
 - Releases (e.g. pull requests titled `Version Packages`)
 
-A maintainer may suggest an addition to the reviewers team by opening a pull request that modifies [`OWNERS.md`](./OWNERS.md) accordingly. Prospective reviewers are not expected to do this themselves, but should rather ask a maintainer to sponsor their addition. All of the maintainers and sponsors are called to vote on the addition (see the section below about voting). If the vote passes, the pull request can be approved and merged, and the corresponding addition to the GitHub team can be made.
+A maintainer may suggest an addition to the reviewers' team by opening a pull request that modifies [`OWNERS.md`](./OWNERS.md) accordingly. Prospective reviewers are not expected to do this themselves, but should rather ask a maintainer to sponsor their addition. All of the maintainers and sponsors are called to vote on the addition (see the section below about voting). If the vote passes, the pull request can be approved and merged, and the corresponding addition to the GitHub team can be made.
```
Comment (freben) [inline GOVERNANCE.md:74]: this one can be reverted - `the "reviewers" team` is the intended meaning here

File: GOVERNANCE.md [hunk 3]
```diff
@@ -71,9 +71,9 @@ The following is a non-exhaustive list of types of change, for which a reviewer
...
 
-A reviewer can elect to remove themselves from the reviewers group by opening, or asking a maintainer to open, a pull request that modifies [`OWNERS.md`](./OWNERS.md) accordingly. A maintainer will approve and merge the pull request, and the corresponding removal from the GitHub team can be made.
+A reviewer can elect to remove themselves from the reviewers' group by opening, or asking a maintainer to open, a pull request that modifies [`OWNERS.md`](./OWNERS.md) accordingly. A maintainer will approve and merge the pull request, and the corresponding removal from the GitHub team can be made.
```
Comment (freben) [inline GOVERNANCE.md:76]: same here

File: STYLE.md [hunk 4]
```diff
@@ -131,7 +131,7 @@ We use [API Extractor](https://api-extractor.com/pages/overview/demo_docs/) to g
 
 The doc comments are of the good old `/** ... */` format, with tags of the format `@<tag>` used to mark various things. The [TSDoc website](https://tsdoc.org/) has a good index of all available tags.
 
-There are a few things to pay attention to make the documentation show up in a nice way on the website...
+There are a few things to pay attention to to make the documentation show up in a nice way on the website...
```
Comment (freben) [inline STYLE.md:134]: maybe even make this "to, in order to" :) so many _to_!

File: docs/getting-started/configuration.md [hunk 5]
```diff
@@ -191,7 +191,7 @@ components: {
 That should be it. You can stop your Backstage App. When you start it again and
 go to your Backstage portal in your browser, you should have your login prompt!
 
-To learn more about Authentication in Backstage, there's the following docs you
+To learn more about Authentication in Backstage, here are the following docs you
```
Comment (freben) [inline docs/getting-started/configuration.md:194]:
```suggestion
To learn more about Authentication in Backstage, here are some docs you
```

the original was kinda correct but using language a bit loosely - the fixed version introduced another oddity though

**Review by freben — COMMENTED** [review]
Thank you very much for doing this! Now that we are here, sorry but I have a few little nits to add :)

**Comment by sakkshm26** [comment]
Thank you for reviewing the PR @jhaals @freben. I have added the required changes.

**Review by freben — APPROVED** [review]
🥳  nice!

**Pull request merged**